platforms/documentation/docs/src/docs/userguide/authoring-builds/build_environment.adoc

```diff
@@ -95,14 +95,12 @@ $ ./gradlew exampleTask -Dorg.gradle.java.home=/path/to/your/java/home --scan
 .gradle.properties
 ----
 org.gradle.java.home=/path/to/your/java/home
-systemProp.http.proxyHost=host.org
 ----
 3. Environment Variable
 +
 [source,text]
 ----
 $ export JAVA_HOME=/path/to/your/java/home
-$ export JAVA_OPTS="-Xmx18928m -XX:+HeapDumpOnOutOfMemoryError -Dfile.encoding=UTF-8 -Djava.awt.headless=true -Dkotlin.daemon.jvm.options=-Xmx6309m"
 ----
 
 == The `gradle.properties` file
```

Comment:
I really like this new section as it really brings everything together. One thing that I think would make a great addition here is mentioning that the `gradle.properties` file can be located in `GRADLE_USER_HOME`, the root project directory, or `GRADLE_HOME`. It is sort of mentioned above in the _Priority for configurations_ section, but you could potentially go into a little more detail here.

Reply:
Added. Great suggestion.